tests/index.ts

```diff
@@ -337,497 +350,502 @@ describe('isSameMinute, isSameHour, isSameDay, isSameMonth, isSameYear', () => {
 
-    expect(getRelativeDate(date)).to.equal('5 дней');
+    expect(formatDistanceToNow(date)).to.equal('5 дней');
   });
 
   it('returns interval string for 1 month', () => {
     const date = new Date();
     date.setMonth(date.getMonth() - 1);
 
-    expect(getRelativeDate(date)).to.equal('месяц');
+    expect(formatDistanceToNow(date)).to.equal('около 1 месяца');
+    expect(formatDistanceToNowStrict(date)).to.equal('1 месяц');
   });
 
   it('returns interval string for 2 months', () => {
     const date = new Date();
     date.setMonth(date.getMonth() - 2);
 
-    expect(getRelativeDate(date)).to.equal('2 месяца');
+    expect(formatDistanceToNowStrict(date)).to.equal('2 месяца');
   });
 
   it('returns interval string for 5 months', () => {
     const date = new Date();
     date.setMonth(date.getMonth() - 5);
 
-    expect(getRelativeDate(date)).to.equal('5 месяцев');
+    expect(formatDistanceToNowStrict(date)).to.equal('5 месяцев');
   });
 
   it('returns interval string for 1 year', () => {
     const date = new Date();
     date.setFullYear(date.getFullYear() - 1);
 
-    expect(getRelativeDate(date)).to.equal('год');
+    expect(formatDistanceToNow(date)).to.equal('около 1 года');
+    expect(formatDistanceToNowStrict(date)).to.equal('1 год');
   });
 
   it('returns interval string for 2 years', () => {
     const date = new Date();
     date.setFullYear(date.getFullYear() - 2);
 
-    expect(getRelativeDate(date)).to.equal('2 года');
+    expect(formatDistanceToNowStrict(date)).to.equal('2 года');
   });
 
   it('returns interval string for 5 years', () => {
     const date = new Date();
     date.setFullYear(date.getFullYear() - 5);
 
-    expect(getRelativeDate(date)).to.equal('5 лет');
+    expect(formatDistanceToNowStrict(date)).to.equal('5 лет');
   });
 });
 
 describe('getUtcOffset', () => {
   it('returns UTC offset in hours', () => {
-    expect(getUtcOffset(new Date(2020, 0, 1))).to.equal(+localUTCOffset);
+    const timezone = format(new Date(), 'xxx');
+    expect(millisecondsToHours(getTimezoneOffset(timezone))).to.equal(+localUTCOffset);
   });
 });
 
 describe('getUnixTimestamp', () => {
   it('returns unix timestamp from Date instance', () => {
-    expect(getUnixTimestamp(new Date(2020, 0, 1))).to.equal(newYear2020UnixTime);
+    expect(getUnixTime(new Date(2020, 0, 1))).to.equal(newYear2020UnixTime);
   });
 
   it('returns unix timestamp from timestamp in ms', () => {
-    expect(getUnixTimestamp(newYear2020UnixTime * 1e3)).to.equal(newYear2020UnixTime);
+    expect(getUnixTime(newYear2020UnixTime * 1e3)).to.equal(newYear2020UnixTime);
   });
 
-  it('returns unix timestamp from timestamp in s', () => {
-    expect(getUnixTimestamp(newYear2020UnixTime)).to.equal(newYear2020UnixTime);
+  it.skip('returns unix timestamp from timestamp in s', () => {
+    // AssertionError: expected 1577811 to equal 1577811600
+    expect(getUnixTime(newYear2020UnixTime)).to.equal(newYear2020UnixTime);
```

Comment:
So, date-fns does not check whether the passed time is Unix timestamp or not. It just divides the passed value by 1000, I guess.